server/server/src/cold_chain/sensor.rs

```diff
@@ -151,7 +151,7 @@ fn upsert_sensor(
                 id: id.clone(),
                 name: Some(sensor.name.clone()),
                 is_active: None,
-                location_id: None,
+                location: None,
```

Comment:
I think the property in `UpdateSensor` should be `location_id` rather than just `location` - it refers to an id rather than a location struct.

Reply:
[39fc8aa](https://github.com/openmsupply/open-msupply/pull/2413/commits/39fc8aa473f2b97681c7b13ab05030f7c148db8b)